consensus/src/dag/anchor_election.rs

```diff
@@ -8,3 +8,21 @@ pub trait AnchorElection {
 
     fn commit(&mut self, round: Round);
 }
+
+pub struct RoundRobinAnchorElection {
+    validators: Vec<Author>,
+}
+
+impl RoundRobinAnchorElection {
+    pub fn new(validators: Vec<Author>) -> Self {
+        Self { validators }
+    }
+}
+
+impl AnchorElection for RoundRobinAnchorElection {
+    fn get_anchor(&self, round: Round) -> Author {
+        self.validators[round as usize % self.validators.len()]
```

Comment:
This is not actually round-robin, because its skipping validators. Works in the good case with Shoal, but not if an anchor doesn't direct commit?

Reply:
you mean round robin with step of 2?

Reply:
yes